### ISSUE TITLE
Bitrix Constructor Error

### ISSUE THREAD

**Issue by thiagows2** [2BAD/bitrix#53]
When attempting to call the Bitrix constructor in my code, I consistently encounter the following error message:

```
TypeError: Bitrix is not a function
    at Object.<anonymous> (/Users/work/workspace/get_leads.js:2:16)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:82:12)
    at node:internal/main/run_main_module:23:47
```

The relevant code snippet is as follows (I have also attempted using import, but encountered the same error):
`get_leads.js`
```javascript
const Bitrix = require('@2bad/bitrix');

const bitrix = Bitrix('https://PORTAL_NAME.bitrix24.ru/rest', 'ACCESS_TOKEN');
```

I appreciate any assistance in resolving this issue.

> Node.js version: v19.0.1

**Comment by friench** [comment]
I have such a problem.

For the decision created the module an interlayer `bitrixClient.js`:
```js
const { default: Bitrix } = require('@2bad/bitrix');
module.exports = Bitrix;
```

And I connected Bitrix through it `bitrixTest.ts`:
```js
import Bitrix from '../utils/bitrixClient';

const bitrix = Bitrix('https://PORTAL_NAME.bitrix24.ru/rest', 'ACCESS_TOKEN')

// Get deal
bitrix.deals.get('77')
  .then(({ result }) => {
    // Get typed payload
    const { TITLE } = result // string
    console.log(TITLE)
  })
  .catch(console.error)

// Get all deals
bitrix.deals.list({ select: ["*", "UF_*"] })
  .then(({ result }) => {
    const titles = result.map((e) => e.TITLE)
    console.log(titles)
  })
  .catch(console.error)
```

I understand that this is a crooked path, but I have not found another answer yet =(

In my case the environment is:
Node: v18.16.0
Nuxt 3.6.1 with Nitro 2.5.2

@2bad/bitrix I use it on the server side to separate requests from the client from requests to Bitrix.

**Comment by mrhyde** [comment]
Fixed in v2.5.1 thanks to @newsiberian
